packages/base-style/src/sass/fonts.scss

```diff
@@ -0,0 +1,147 @@
+@at-root {
```

Comment:
can we have built-in font instead of fetching from url

Reply:
What's the benefit of having it built-in instead of fetching from Google Web Fonts API?
It's currently more beneficial to keep as is, as Google Web Fonts are widely used, they offer caching benefits and also handles the cases where a fallback is needed. It also handles different implementations of web fonts in different browsers.

Referring to this excerpt here: `Fonts served by the Google Fonts API are automatically compressed for a faster download, and once downloaded are cached in the browser and reused by any other web page that uses the Google Fonts API.` which is taken from this [link]('https://developers.google.com/fonts/faq#will_web_fonts_slow_down_my_page').